geo/CHANGES.md

```diff
@@ -55,9 +55,6 @@
   * <https://github.com/georust/geo/pull/629>
 * Implement `Centroid` algorithm on `Geometry` and its remaining variants.
   * <https://github.com/georust/geo/pull/629>
-
```

Comment:
👀

Reply:
There were two 0.17.1 headings in the changelog. I assume that was an accident so I'm merging them together.